sw/device/tests/sram_ctrl_sleep_sram_ret_contents_test.c

```diff
@@ -205,8 +205,8 @@ bool test_main(void) {
 
   // Turn off the AON timer hardware completely before exiting.
   aon_timer_testutils_shutdown(&aon_timer);
-  if (kDeviceType == kDeviceSimDV) {
-    CHECK_DIF_OK(dif_rstmgr_software_device_reset(&rstmgr));
-  }
+  // if (kDeviceType == kDeviceSimDV) {
```

Comment:
Is this change related? Or should it be dropped?

Reply:
This is actually an interesting bug that we need to look into.
It is present in another test also.
See #16597.

But same question from my side - is this hunk related to this PR?

Reply:
oh no, I was debugging 2 issues in a same branch and accidentally got this in. Let me remove this one.